README.md

```diff
@@ -149,11 +149,22 @@ Instances of `Record` and `Tuple` are immutable, so their equality works like th
 
 ```js
 assert(#{ a: 1 } === #{ a: 1 });
-assert(Object(#{ a: 1 }) !== Object(#{ a: 1 }));
+assert(#[1, 2] === #[1, 2]);
+```
+
+This is distinct from how equality works for JS objects. Strict comparison of objects will observe that each object is distinct: 
+
+```js
 assert({ a: 1 } !== { a: 1 });
+assert(Object(#{ a: 1 }) !== Object(#{ a: 1 }));
+assert(Object(#[1, 2]) !== Object(#[1, 2]));
 ```
 
-This is distinct from how equality works for JS objects: `assert({} !== {});`. Strict comparison of objects will observe that each object is distinct.
+Insertion order of record keys does not affect equality of records:
```

Comment:
I’d still like an example of ordering mattering for tuples here.

Reply:
@mheiber ^

Reply:
or alternatively I can add the example, up to you @mheiber

Reply:
Sure @rickbutton

I was not expecting insertion order to matter for tuples.

That is, I would expect `#[...#[1]]` to equal `#[1]`

Reply:
That’s not insertion order tho.

It’d be, would [1, 2] equal [2, 1]?